app/src/main/java/com/example/home/stream/StreamHomeDetailsActivity.java

```diff
@@ -0,0 +1,13 @@
+package com.example.home.stream;
+
+import android.os.Bundle;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+public class StreamHomeDetailsActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
```

Comment:
This time is okay to add it here. But please try to only add the code that you are using in the PR